gradle.properties

```diff
@@ -1,20 +0,0 @@
-## Project-wide Gradle settings.
-#
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-#
-# Specifies the JVM arguments used for the daemon process.
```

Comment:
why have you removed this all?
You don't need to send this file in pull request.